scylla-cql/src/frame/value.rs

```diff
@@ -41,20 +42,206 @@ pub enum MaybeUnset<V: Value> {
     Set(V),
 }
 
-/// Wrapper that allows to send dates outside of NaiveDate range (-262145-1-1 to 262143-12-31)
+/// Native date representation that allowes for a bigger range of dates (-262145-1-1 to 262143-12-31)
 /// Days since -5877641-06-23 i.e. 2^31 days before unix epoch
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
-pub struct Date(pub u32);
+pub struct CqlDate(pub u32);
 
 /// Wrapper used to differentiate between Time and Timestamp as sending values
 /// Milliseconds since unix epoch
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
-pub struct Timestamp(pub Duration);
+pub struct CqlTimestamp(pub i64);
 
 /// Wrapper used to differentiate between Time and Timestamp as sending values
 /// Nanoseconds since midnight
 #[derive(Clone, Copy, PartialEq, Eq, Debug)]
```

Comment:
Since original docstrings don't make a lot of sense with this implementation, I decided to rewrite the entire docstring, rather than just update the type name